Libplanet/Net/NetMQTransport.cs

```diff
@@ -541,6 +568,12 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
                     ReplyMessage(differentVersion);
                     _logger.Debug("Message from peer with different version received.");
                 }
+                catch (InvalidTimestampException ite)
+                {
+                    const string logMsg = "The received message is stale. " +
+                              "(timestamp: {Timestamp}, lifespan: {Lifespan}, current: {Current})";
+                    _logger.Debug(logMsg, ite.CreatedOffset, ite.Lifespan, ite.CurrentOffset);
```

Comment:
Is it necessary to make a separate message string?

Reply:
This was because line restriction. Single argument spans multiple lines against the rule.